src/hardhat/utils/index.ts

```diff
@@ -30,69 +37,42 @@ export function getW3fDetails(w3fRootDir: string) {
       } else return;
 
       // get userArgs
-      try {
-        const userArgsJsonString = fs.readFileSync(userArgsJsonPath, "utf8");
-        const userArgs = JSON.parse(userArgsJsonString);
-
-        w3fDetails[file] = {
-          path: indexPath,
-          userArgs,
-          storage: {},
-          secrets: {},
-        };
-      } catch (error) {
-        console.error(
-          `Error reading userArgs.json for ${file}: ${error.message}`
-        );
-        w3fDetails[file] = {
-          path: indexPath,
-          userArgs: {},
-          storage: {},
-          secrets: {},
-        };
-      }
+      if (fs.existsSync(userArgsJsonPath)) {
+        try {
+          const userArgsJsonString = fs.readFileSync(userArgsJsonPath, "utf8");
+          userArgs = JSON.parse(userArgsJsonString);
+        } catch (error) {
+          console.error(
+            `Error reading userArgs.json for ${file}: ${error.message}`
+          );
+        }
+      } else console.warn(`${WARN} userArgs.json not found\n`);
```

Comment:
I don't think this should be a warning because some web3 functions don't require userArgs

Reply:
[780579f](https://github.com/gelatodigital/web3-functions-sdk/pull/22/commits/780579feb90f9799a7d3d816607ff161cbf6598c)